src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -208,20 +217,18 @@ def lower_to_language_level(self):
         # pylint: disable=import-outside-toplevel
         from psyclone.psyGen import InvokeSchedule
 
-        if not self._node_lowered:
-            # We must generate a list of all of the fields accessed by OpenACC
-            # kernels (calls within an OpenACC parallel or kernels directive)
-            # 1. Find all parallel and kernels directives. We store this list
-            # for later use in any sub-class.
-            self._acc_dirs = self.ancestor(InvokeSchedule).walk(
-                    (ACCParallelDirective, ACCKernelsDirective))
-            # 2. For each directive, add the fields used by the kernels it
-            # contains (as given by kernel_references) and add it to our set.
-            # TODO GOcean grid properties are duplicated in this set under
-            # different names (the OpenACC deep copy support should spot this).
-            for pdir in self._acc_dirs:
-                self._sig_list.update(pdir.kernel_references)
-            self._node_lowered = True
+        # We must generate a list of all of the fields accessed by OpenACC
+        # kernels (calls within an OpenACC parallel or kernels directive)
+        # 1. Find all parallel and kernels directives. We store this list
+        # for later use in any sub-class.
+        self._acc_dirs = self.ancestor(InvokeSchedule).walk(
```

Comment:
Done.

Let me know if [this](https://github.com/stfc/PSyclone/blob/323bed717e1cec63caea0f9a1b9f4eda08e5497c/src/psyclone/transformations.py#L2274-L2275) suffices, otherwise I can add a comment here too.